cmd/mesos_metrics_test.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import "testing"
+import "encoding/json"
+
+func parseon() string {
```

Comment:
This isn't used anywhere, so I'm guessing it should be deleted. Otherwise, you're ignoring an error in the `json.Unmarshal`.